src/components/Cluster/ClusterDetail/NodesRunning.tsx

```diff
@@ -18,7 +18,7 @@ const NodesRunning: FC<INodesRunningProps> = ({
   numNodePools,
   workerNodesRunning,
 }) => {
-  if (workerNodesRunning === 0 && isClusterCreating) {
+  if (isClusterCreating) {
```

Comment:
Also small change here, we should always display the `Nodes are not ready` while a cluster is creating.